lapack.go

```diff
@@ -122,6 +122,12 @@ const (
 	EigBoth EigComp = 'I'
 )
 
+// JobEV specifies whether eigenvectors will be computed in Dsyev.
```

Comment:
I think these all would be better as `EVJob` , `EVCompute`, `EVSide`, etc.. It matches both `SVDJob` and `EigDecomp`.